sqlglot/dialects/snowflake.py

```diff
@@ -746,6 +745,7 @@ class Parser(parser.Parser):
             "PARSE_URL": lambda args: exp.ParseUrl(
                 this=seq_get(args, 0), permissive=seq_get(args, 1)
             ),
+            "REGEXP_COUNT": exp.RegexpCount.from_arg_list,
```

Comment:
This is redundant right? Due to the `PascalCase -> SNAKE_UPPER_CASE` convention.